protocol/0045-DSRC-data_sourcing.md

```diff
@@ -246,3 +246,6 @@ Vega should reject any data source tx that is not explicitly required, so this w
 1. After trading termination has been triggered the trading terminated data source is no longer active (assuming it is not used anywhere else) and data from that source is no longer processed (<a name="0045-DSRC-019" href="#0045-DSRC-019">0045-DSRC-019</a>)
 1. After settlement data has been received and the market has settled, the settlement data source is no longer active (assuming it is not used anywhere else) and data from that source is no longer processed (<a name="0045-DSRC-020" href="#0045-DSRC-020">0045-DSRC-020</a>)
 1. Where the same oracle definition is used for trading terminated and settlement, and data has been received. Trading is terminated and the market is settled with the same data event (a single message does both). The data source is no longer active (assuming it is not used anywhere else) and data from that source is no longer processed. (<a name="0045-DSRC-021" href="#0045-DSRC-021">0045-DSRC-021</a>)
+1. It should be possible to update the market termination spec from external to internal and prove that the original termination key and filter no longer supports termination and that the updated market termination key and filter supports termination. This should apply to filter key types Timestamp/Boolean and Integer. (<a name="0045-DSRC-022" href="#0045-DSRC-022">0045-DSRC-022</a>)
+1. It should be possible to update the market termination spec from internal to external and prove that the original termination key and filter no longer supports termination and that the updated market termination key and filter supports termination. This should apply to filter key types Timestamp/Boolean and Integer. (<a name="0045-DSRC-023" href="#0045-DSRC-023">0045-DSRC-023</a>)
+1. It should not be possible to update the market settlement spec from external to internal type (<a name="0045-DSRC-024" href="#0045-DSRC-024">0045-DSRC-024</a>)
```

Comment:
```suggestion
1. It should not be possible to update the market settlement spec from external to internal type. (there is no requirement to prevent making settlement data an internal source, it's just currently true that no internal sources emit data that could be used for settlement so it should in practice not be possible at the moment). (<a name="0045-DSRC-024" href="#0045-DSRC-024">0045-DSRC-024</a>)
```